apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerDbTypeTests.ValueConversions.cs

```diff
@@ -113,6 +114,16 @@ private static IEnumerable<SpannerNumeric> GetSpannerNumericsForArray()
             yield return SpannerNumeric.MaxValue;
         }
 
+        private static IEnumerable<string> GetJsonStringsForArray()
+        {
+            // These are not well-formatted JSON strings, but that is not checked
```

Comment:
Again, I'd far rather use valid data if possible. (It's not clear what you mean by it preventing problems with tests that serialize struct values to JSON... sure, we might end up with some escaping going on, but I'd like to see that in tests anyway.)

Reply:
I've changed the input and expected values to valid JSON strings (including the necessary escaping).